MCKIN-7653 instructions popup for IE-v2

murad-hubib opened the pull request:
- Added instructions pop-up for version V2
- Clicking "Start Exercise" button should close the popup

> It's a default popup and user will see it every time so there is no step/s to produce this feature

Ticket: [MCKIN-7653](https://edx-wiki.atlassian.net/browse/MCKIN-7653)

edx-webhook commented:
Thanks for the pull request, @murad-hubib! I've created [OSPR-2506](https://openedx.atlassian.net/browse/OSPR-2506) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation.     If you like, you can add yourself to the [AUTHORS](https://github.com/edx-solutions/xblock-image-explorer/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

kaizoku commented on image_explorer/templates/html/image_explorer.html:
```diff
@@ -36,6 +37,15 @@ <h2>{{title}}</h2>
+                <h3>Instructions</h3>
+                <p>Click each step in the loop to learn more about the problem solving approach. When you are finished, click the forward arrow to continue.</p>
+                <div class="tutorial-video">&nbsp;</div>
+                <a href="javascript:void(0)" id="hide-me">Start Excercise</a>
```
Does this link need the `href` attribute? I think if we leave this off it should have the same effect were there no javascript handler.

Also could we change the ID for this to be a bit more specific. Maybe something like `#hide-tutorial`, or `#hide-instructions` ?

murad-hubib replied:
Thanks @kaizoku for the prompt review. I have removed the "href" attribute and changed the ID to "hide-tutorial"

Can you please merge the code as I don't have the rights?

murad-hubib replied:
@kaizoku Please wait as I have got some feedback from Olivia and I have to do some tweaks in colors. I will let you know when to merge the code. Thanks

kaizoku commented on image_explorer/templates/html/image_explorer.html:
```diff
@@ -36,6 +37,15 @@ <h2>{{title}}</h2>
+                <h3>Instructions</h3>
+                <p>Click each step in the loop to learn more about the problem solving approach. When you are finished, click the forward arrow to continue.</p>
+                <div class="tutorial-video">&nbsp;</div>
+                <a id="hide-tutorial">Start Excercise</a>
```
This should be "Exercise". I think there's an extra 'c' in there.
Sorry I didn't notice this before.

murad-hubib commented:
@kaizoku Thanks for poinnting out the typo. I have fixed it.

And I have addressed the color/theme changes and have pushed in this PR.

You can now merge the PR

kaizoku commented:
:+1:

- [✓] I tested this: (deployed in devstack and tested changes before and after patch)
- [✓] I read through the code
- [✓] I checked for accessibility issues
- [✓] Includes documentation
